### ISSUE TITLE
fix: Screen readers do not announce options in Select listbox

### ISSUE THREAD

**Issue by daviddossett** [microsoft/fast#5243]
# 🐛 Bug Report

When using keyboard navigation to interact with a select component, options in the list are not announced by the screen reader. Also described in #4514.

## 💻 Repro or Code Sample

1. Active the screen reader on your computer.
2. Focus on a `fast-select` component. The [fast-select example](https://explore.fast.design/components/fast-select) in the component explorer is a reliable repo. 
3. Hit enter to open the listbox
4. Use arrow keys to scan through the options

## 🤔 Expected Behavior

The screen reader should announce each value as the user focuses on it. Native select/options elements do this out of the box. Interestingly enough, the [fast-combobox](https://explore.fast.design/components/fast-combobox) works as expected.

## 😯 Current Behavior

No change is announced when focusing on option elements.

## 🌍 Your Environment

* OS & Device: macOS and Windows 11
* Browser: Microsoft Edge
* Version: 94.0.992.31

cc @hawkticehurst
